monitoring/src/main/scala/akkaviz/serialization/ClassInspector.scala

```diff
@@ -35,7 +39,7 @@ object ClassInspector {
 
   private val rm = scala.reflect.runtime.currentMirror
 
-  case class UnableToInspectField(t: Throwable)
+  case object UnableToInspectField
 
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Method is longer than 50 lines](https://www.codacy.com/app/lustefaniak/akka-viz/file/2215348010/issues/source?bid=3121282&fileBranchId=3172196#l44)